docs/specs/metadata.yml

```diff
@@ -28,8 +28,107 @@ inputs:
     # Title
 outputs:
   docs/a.json: |
-    { "ms.custom-metadata": null, "author":"xxx", "uid": null, "breadcrumb_path": "/breadcrumb_path", "title": "Title"}
+    { 
+      "ms.custom-metadata": null, 
+      "author":"xxx", 
+      "uid": null, 
+      "breadcrumb_path": "/breadcrumb_path", 
+      "title": "Title",    
+    }
+---
+# output _xrefmap per file to structure json
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    ---
+    uid: abc
+    breadcrumb_path: "/breadcrumb_path"
+    author: xxx
+    ---
+    # Title
+outputs:
+  docs/a.json: |
+    {      
+      "author":"xxx", 
+      "uid": "abc", 
+      "breadcrumb_path": "/breadcrumb_path", 
+      "title": "Title",  
+      "_xrefmap": {
+          "abc":{
+            "href": "/docs/a",
+            "name": "abc",
+            "uid": "abc",
+            "specType": "uid",
+          }
+      }
+    }
+  .xrefmap.json: | 
+    {
+      "references":[
+        {"uid": "abc", "name": "abc" ,"specType": undefined },       
+      ]
+    }
+---
+# In continue build, output xrefmap to page metadata but not page data
+continueBuildOnly: true
+inputs:
+  docfx.yml: |
+    outputType: pageJson
+  docs/a.md: |
+    ---
+    uid: abc
+    breadcrumb_path: "/breadcrumb_path"
+    author: xxx
+    ---
+    # Title
+outputs:  
+  docs/a.raw.page.json: | 
+    { 
+      "rawMetadata": {
+        "xrefmap": undefined
+      }
+    }
+  docs/a.mta.json: |
+    {       
+      "xrefmap": "[{\"uid\":\"abc\",\"href\":\"/docs/a\",\"specType\":\"uid\",\"name\":\"abc\"}]"
```

Comment:
>/docs/a\ [](http://example.com/codeflow?start=47&length=8)

we should generate the absolute path `href`, should be the same with the value inside `.xrefmap.json`